docs/form.component.md

```diff
@@ -235,6 +235,197 @@ also implement your own custom validators to replace or extend the set. See the
 
 ### Common scenarios
 
+#### Render form by using form definition JSON
+
+Below is a sample form definition JSON.
+
+```json
+    {
+            "id": 1001,
+            "name": "ISSUE_FORM",
+            "processDefinitionId": "ISSUE_APP:1:2504",
```

Comment:
Do we need those informations?
processDefinitionId
processDefinitionName
processDefinitionKey
taskDefinitionKey
taskId

Reply:
Even though these properties are not required to render the form, these are part of standard API responses.

If the form pertains to a process, then all of these properties are part of the [task-form](https://docs.alfresco.com/activiti/docs/dev-guide/1.5.0/#_task_form) API response and if the task does not pertain to process, then `taskId` alone is available in the response.